src/main/java/frc/trigon/robot/subsystems/pitcher/Pitcher.java

```diff
@@ -0,0 +1,114 @@
+                .angularVelocity(Units.RotationsPerSecond.of(masterMotor.getSignal(TalonFXSignal.VELOCITY)))
+                .voltage(Units.Volts.of(masterMotor.getSignal(TalonFXSignal.MOTOR_VOLTAGE)));
+    }
+
+    @Override
+    public void setBrake(boolean brake) {
+        masterMotor.setBrake(brake);
+        followerMotor.setBrake(brake);
+    }
+
+    @Override
+    public void stop() {
+        masterMotor.stopMotor();
+    }
+
+    @Override
+    public void periodic() {
+        masterMotor.update();
+        updateMechanism();
+    }
+
+    @Override
+    public void drive(Measure<Voltage> voltageMeasure) {
+        masterMotor.setControl(voltageRequest.withOutput(voltageMeasure.in(Units.Volts)));
+    }
+
+    @Override
+    public SysIdRoutine.Config getSysIdConfig() {
+        return PitcherConstants.SYSID_CONFIG;
+    }
+
+    public Rotation2d getCurrentPitch() {
+        return Rotation2d.fromRotations(masterMotor.getSignal(TalonFXSignal.POSITION));
+    }
+
+    public Rotation2d getTargetPitch() {
+        return targetPitch;
+    }
+
+    public boolean atTargetPitch() {
+        return Math.abs(masterMotor.getSignal(TalonFXSignal.POSITION) - targetPitch.getRotations()) < PitcherConstants.PITCH_TOLERANCE.getRotations();
+    }
+
+    void reachTargetPitchFromShootingCalculations() {
+        targetPitch = shootingCalculations.getTargetShootingState().targetPitch();
+        setTargetPitch(targetPitch);
+    }
+
+    void setTargetPitch(Rotation2d targetPitch) {
+        this.targetPitch = targetPitch;
+        masterMotor.setControl(positionRequest.withPosition(targetPitch.getRotations()));
+    }
+
+    private void updateMechanism() {
+        PitcherConstants.PITCHER_AND_AMP_ALIGNER_MECHANISM.updateFirstJoint(
+                getCurrentPitch(),
+                Rotation2d.fromRotations(masterMotor.getSignal(TalonFXSignal.CLOSED_LOOP_REFERENCE))
+        );
+        final Pose3d pitcherPose = calculatePitcherComponentPose();
+        Logger.recordOutput("Poses/Components/PitcherPose", pitcherPose);
+        Logger.recordOutput("Poses/Components/AmpAlignerPose", calculateAmpAlignerComponentPose(pitcherPose));
+    }
+
+    private Pose3d calculatePitcherComponentPose() {
+        final Transform3d pitcherTransform = new Transform3d(
+                new Translation3d(),
+                new Rotation3d(0, getCurrentPitch().getRadians(), 0)
+        );
+        return PitcherConstants.PITCHER_VISUALIZATION_ORIGIN_POINT.transformBy(pitcherTransform);
+    }
+
+    private Pose3d calculateAmpAlignerComponentPose(Pose3d pitcherPose) {
+        final Transform3d ampAlignerTransform = new Transform3d(
+                new Translation3d(),
+                new Rotation3d(0, RobotContainer.AMP_ALIGNER.getCurrentAngle().getRadians(), 0)
+        );
+        return pitcherPose.transformBy(ampAlignerTransform).transformBy(AmpAlignerConstants.PITCHER_TO_AMP_ALIGNER);
```

Comment:
transform the pitcherComponentPose (rename) by the PITCHER_TO_AMP_ALIGNER, and then use the previously mentioned `addPitch` function to add the amp aligner angle to everything.